mozilla_schema_generator/__main__.py

```diff
@@ -144,7 +144,7 @@ def write_schema(repo, repo_id, config, out_dir, split, pretty, generic_schema):
     dump_schema(schemas, out_dir.joinpath(repo_id), pretty)
 
 
-def dump_schema(schemas, out_dir, pretty):
+def dump_schema(schemas, out_dir, pretty, *, version=1):
```

Comment:
I haven't seen `*` used like this before, this is the same pattern as `*args, **kwargs`? Why not omit the `*`?

Reply:
This is new to me too so I looked it up. [PEP 3102](https://www.python.org/dev/peps/pep-3102/) describes it in detail. Basically it forces anything after the `*` to be a keyword parameter and doesn't allow any more positional args aside from the ones defined like `*args` would do.